server/views/applications/before-you-start.njk

```diff
@@ -9,7 +9,7 @@
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
       <h1>Apply for a CAS-2 placement</h1>
-      <p>Community Accommodation Service Tier 2 (CAS-2) provides housing for people who do not have a suitable address for the term of their licence or Bail Order.</p>
+      <p>Short-Term Accommodation (CAS-2/Tier-2) provides housing for people who do not have a suitable address for the term of their licence or Bail Order.</p>
```

Comment:
I had to make a decision here given "Tier-2" already existed. I'm not convinced about adding it on as another part of the brackets but it felt like a safer stance to take?

We could of course remove it in favour of "Short-Term Accommodation (CAS-2)" or something else though as this page appears to be a start page it felt like a good place to relate to as many people as possible.

Reply:
@chaneldiepmoj what do you think?

Reply:
[Conversation continued in the Slack thread](https://mojdt.slack.com/archives/C04T6SC31HA/p1704821481047049?thread_ts=1704812197.680799&cid=C04T6SC31HA)